source/slang/slang.cpp

```diff
@@ -3178,6 +3246,43 @@ ISlangUnknown* Module::getInterface(const Guid& guid)
     return Super::getInterface(guid);
 }
 
+// One implementation under ComponentType that takes entryPointIndex/targetIndex
+// One per subtype without
```

Comment:
These comments were developer notes during the process of implementing the feature, but aren't really useful to anybody reading the code now.